src/olympia/addons/tests/test_models.py

```diff
@@ -1280,18 +1280,18 @@ def test_can_request_review_rejected(self):
         addon.latest_version.files.update(status=amo.STATUS_DISABLED)
         assert addon.can_request_review() == ()
 
-    def check(self, status, exp, kw={}):
+    def check(self, status, exp, no_prelim=False, kw={}):
         addon = Addon.objects.get(pk=3615)
         changes = {'status': status, 'disabled_by_user': False}
         changes.update(**kw)
         addon.update(**changes)
-        assert addon.can_request_review() == exp
+        assert addon.can_request_review(no_prelim) == exp
```

Comment:
Actually, I know it was already there but I really hate that `check` method. By looking at the calls below you have no idea what it's doing.

I suggest changing its signature to use keyword arguments and use those keyword arguments in every call below. Something like this:

```
def check(self, status=None,  expected_statuses=None, disallow_preliminary_review=False,
          extra_update_kwargs={}):
    if status is None or  expected_statuses is None:
        raise ImproperlyConfiguredError
    addon = Addon.objects.get(pk=3615)
    changes = {'status': status, 'disabled_by_user': False}
    changes.update(**extra_update_kwargs)
    addon.update(**changes)
    assert addon.can_request_review() == expected_statuses
    assert addon.can_request_review(disallow_preliminary_review) ==  expected_statuses
```